py-polars/src/dataframe.rs

```diff
@@ -1248,6 +1248,11 @@ impl PyDataFrame {
         df.into()
     }
 
+    pub fn approx_unique(&self) -> PyResult<Self> {
```

Comment:
I only want this accessable on the expression API.

Reply:
done